chipsec/library/returncode.py

```diff
@@ -105,6 +102,10 @@ def getReturnCode(self, result: int) -> int:
         self.resetReturnCodeValues()
         return ret_value
 
+def generate_hash_id(className: str) -> str:
```

Comment:
Should update to return int:
```suggestion
def generate_hash_id(className: str) -> int:
```